src/Sarif.Viewer.VisualStudio/SarifViewerPackage.cs

```diff
@@ -28,21 +31,19 @@ namespace Microsoft.Sarif.Viewer
     [InstalledProductRegistration("#110", "#112", "2.0 beta", IconResourceID = 400)] // Info on this package for Help/About
     [Guid(SarifViewerPackage.PackageGuidString)]
     [ProvideMenuResource("Menus.ctmenu", 1)]
-    [ProvideEditorExtension(typeof(SarifEditorFactory), ".sarif", 128)]
-    [ProvideToolWindow(typeof(SarifToolWindow), Style = VsDockStyle.Tabbed, Window = "3ae79031-e1bc-11d0-8f78-00a0c9110057")]
+    [ProvideToolWindow(typeof(SarifToolWindow), Style = VsDockStyle.Tabbed, Window = "3ae79031-e1bc-11d0-8f78-00a0c9110057", Transient = true)]
     [ProvideService(typeof(SLoadSarifLogService))]
     [ProvideService(typeof(SCloseSarifLogService))]
     public sealed class SarifViewerPackage : AsyncPackage
     {
-        public static DTE2 Dte;
-        public static IServiceProvider ServiceProvider;
-
-        private SarifEditorFactory _sarifEditorFactory;
+        private DTE2 Dte;
```

Comment:
Is this used anymore?

Reply:
Nope. Neither one actually. Which is goodness. Removing both.

---
In reply to: [475869907](https://github.com/microsoft/sarif-visualstudio-extension/pull/179#discussion_r475869907) [](ancestors = 475869907)